packages/tests/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tests",
+  "name": "@trpc/tests",
```

Comment:
if we can get away with not naming this @trpc it makes our lives a lot easier.

all the install filters use @trpc/* and that would make this package match that too => unnecessary package installs in all examples and www

Reply:
I'm unsure if this even helps tbh.... but won't turborepo just cache it anyway?

Reply:
not the dependency installation. we can add another filter that removes this

```bash
pnpm --filter="@trpc/*" --filter="\!trpc@tests" install
```

Reply:
Cool! Feel free to commit on this branch if you think this is the way :)